pci_dss_v321/requirement_7.sp

```diff
@@ -0,0 +1,81 @@
+benchmark "pci_dss_v321_requirement_7" {
+  title       = "Requirement 7 - Restrict access to cardholder data by business need-to-know"
+  description = "To ensure critical data can only be accessed by authorized personnel, systems and processes must be in place to limit access based on need to know and according to job responsibilities. Need to know is when access rights are granted to only the least amount of data and privileges needed to perform a job."
+  children = [
+    benchmark.pci_dss_v321_requirement_7_1,
+    benchmark.pci_dss_v321_requirement_7_2
+  ]
+
+  tags = local.pci_dss_v321_common_tags
+}
+
+benchmark "pci_dss_v321_requirement_7_1" {
+  title = "Limit access to system components and cardholder data to only those individuals whose job requires such access"
+  children = [
+    benchmark.pci_dss_v321_requirement_7_1_1,
+    benchmark.pci_dss_v321_requirement_7_1_2,
+    benchmark.pci_dss_v321_requirement_7_1_3
+  ]
+
+  tags = local.pci_dss_v321_common_tags
+}
+
+benchmark "pci_dss_v321_requirement_7_1_1" {
+  title = "Define access needs for each role, including: - System components and data resources that each role needs to access for their job function - Level of privilege required (for example, user, administrator, etc.) for accessing resources"
+  children = [
+    control.iam_subscription_owner_max_3,
+    control.iam_subscription_owner_more_than_1
+  ]
+
+  tags = merge(local.pci_dss_v321_common_tags, {
+    service = "Azure/ActiveDirectory"
```

Comment:
Should the service be `Azure/IAM`?

Should all the benchmarks refer to the `Azure/IAM` service?

Reply:
Service should be `Azure/ActiveDirectory`